packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/theme/design_system/design_system_typography.dart

```diff
@@ -80,6 +80,9 @@ fontWeight: FontWeight.w400, fontStyle: FontStyle.normal, fontSize: 20.0);
       fontWeight: FontWeight.w400, fontStyle: FontStyle.normal, fontSize: 11.0);
 
   /// App specific typography
+
+  late final fontWeightBold = const TextStyle(fontWeight: FontWeight.bold);
```

Comment:
Please follow the convention or use some of the already implemented props such as `h1Bold18`

Reply:
Done